app/test-pages/amd/getting-diagnosed.njk

```diff
@@ -0,0 +1,144 @@
+{% set title = 'Getting diagnosed | Age-related macular degeneration (AMD)  | Test page' %}
```

Comment:
See comment about page titles, should be 'Page title - Website', we don't include page sections.